tutorial/chapter1.md

```diff
@@ -0,0 +1,96 @@
+<p align="right">
+  <img width="25%" height="25%"src="../images/klepsydra_logo.jpg">
+</p>
+
+# Introduction
+Klepsydra vision API is split into main groups also (as well as Klepsydra API):
+
+* **Application API**
+* **Composition API**
```

Comment:
the vision_ocv subfolder in kpsr-robotics has the classes which form the application api and the composition api -- write those names here.